.github/workflows/build.yml

```
@@ -0,0 +1,238 @@
+name: Continuous FSMP mod build
+
+on:
```

YAMLlint (1.38.0)

[warning] truthy value should be one of [false, true]

(truthy)